sum_types_generator/lib/src/sum_type_spec.dart

```diff
@@ -124,7 +125,7 @@ SumTypeSpec makeSumTypeSpec(Element element, ConstantReader annotation) {
       typeParams: element.typeParameters.map(
         (e) => TypeParamSpec(
           name: e.name,
-          bound: e.bound?.element?.name,
+          bound: e.bound?.element2?.name,
```

Comment:
Analyzer added `element2` deprecating `element` in version 4.6.0
Then, it removed `element` in version 5.0.0
Then, it added `element` again in version 5.2.0, deprecating `element2` this time '-'

In this case, I need to use `element2` if we want to support more versions.
Or I can use `element`, but the minimum supported version will need to be 5.2.0

What do you prefer?

Reply:
I prefer whatever works best for our internal project. If either option is good for the internal project, I prefer the most modern approach (I understand it's [again] just `.element`).